source/appModules/thunderbird.py

```diff
@@ -16,6 +16,21 @@
 
 
 class AppModule(appModuleHandler.AppModule):
+	def _isPopupMenuItem(self, obj):
```

Comment:
please add type hints to these functions

```suggestion
	def _isPopupMenuItem(self, obj: NVDAObject):
```